src/amuse/io/nemobin.py

```diff
@@ -273,147 +282,141 @@ def get_item_header(self):
         elif magic_number == self.reversed_PlurMagic:
             magic_number == self.PlurMagic
             mustswap = True
-            
+
         if not (magic_number == self.SingMagic or magic_number == self.PlurMagic):
             raise base.IoException("Item does not have a valid header")
-        
+
         typecharacter = self.read_string()
         if not typecharacter == TesItem.typecharacter:
             tagstring = self.read_string()
         else:
             tagstring = ''
-        if magic_number ==  self.PlurMagic:
+        if magic_number == self.PlurMagic:
             dim = self.read_array('i')
             if mustswap:
                 dim.byteswap()
             dim = dim.tolist()
         else:
             dim = [1]
-            
+
         return (typecharacter, tagstring, dim, mustswap)
-        
+
     def read_item(self):
         typecharacter, tagstring, dim, mustswap = self.get_item_header()
         if typecharacter is None:
             return None
         result = NemoItemType.new_item(typecharacter, tagstring, dim, mustswap)
         result.read(self)
         return result
-    
+
     def read(self):
         result = OrderedMultiDictionary()
-        
+
         item = self.read_item()
-        while not item is None:
+        while item is not None:
             result[item.tagstring] = item
             item = self.read_item()
-        
+
         return result
-    
+
     def write_magic_number(self, is_plural):
         if is_plural:
             magic_number = self.PlurMagic
         else:
             magic_number = self.SingMagic
-            
+
         x = array.array('h', [magic_number])
-        self.file.write(x.tostring())
-        
-    
+        self.file.write(x.tobytes())
+
     def write_array(self, typetag, data):
         x = array.array(typetag, data)
         x.append(0)
-        self.file.write(x.tostring())
-        
+        self.file.write(x.tobytes())
```

Comment:
why is this?

Reply:
fixes a deprecation warning

Reply:
(or at least some sort of warning that came up... very specific in that it should be fixed this way.)